apps/discussions/app/modules/Comment.js

```diff
@@ -133,13 +152,13 @@ const Comment = ({
           onSave={update}
         />
       ) : (
-        <React.Fragment>
-          <Top author={author} createdAt={createdAt} />
+        <Fragment>
```

Comment:
Not a big deal, but just note that recent react versions allow to omit the `Fragment` part so you can just type it as:
```suggestion
        <>
```